src/instr.c

```diff
@@ -443,7 +450,8 @@ void Gack(VM* vm, VM_Shred shred, Instr instr)
 #else
     fprintf(stdout, ") ");
 #endif
-    for(int j = 0; j < longest - strlen(name); j++)
```

Comment:
I know that this sort of for loop is "legal" in c99 mode, but in practice I tend not to use this feature. I have just declared it outside of the loop.